### DISCUSSION TITLE
New Sensor Options from Atlas Scientific--thoughts?

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#872] [Q&A]
I'm looking ahead a bit towards adding DIY REM Chem to my pool & just saw there are new industrial sensor options from A-S.  They now have ORP  and pH sensors with SMA connectors on them, so no messing around adding connectors & worrying about proper shielding, etc.  I'm guessing I can pull the cables through a cable gland as-is & screw them directly to the Whitebox T3 board with the EZO Adapters on them.  Probably want to include strain relief along the way.  Nice, if true!

However, the pH probe does not include a temperature sensor with the SMA connector.  My question is does this really matter?  Like most/all of you, I  have a water temperature sensor at the pad that will be connected to the poolController via the MegaBas (my controller is still being built--waiting for parts).  Can this temperature probe be used for pH calculations?  If  so, is there anything  I need to do to make temperature available for pH calculations?

As always, thanks for your thoughts & assistance!

****   Update   ****

I think I just answered my own question when re-reading the REM Chem wiki.  To quote the wiki:

"The Atlas industrial pH probe has a built in temperature sensor. The advantage to this is that the temperature calibration for pH is built right into the pH probe. However, if you are using the Atlas Lab or Consumer grade probes you will need to set up a water temperature feed in the software to the probe. This is simply a checkbox in dashPanel that will send the water temperature to the probe so that it can calculate a properly adjusted pH value."

So, if my understanding is correct, getting the "new" versions of the probes may simply connecting everything & per the wiki, the existing temperature probe should provide the data needed in the pH calculation.  The other advantage of the new pH probe is it's a little less expensive than the unit without the SMA connector and temperature probe.  Every little bit helps, right?

**Comment by rstrouse** [comment]
Yep you can feed the water temp to the probe to get the proper calibration.  The big PITA for the other probes is the lack of SMA connectors on them.

**Reply by FlaMike** [reply]
Thank you for confirming I may use the existing temperature probe!

I've been concerned about getting the probe without an SMA connector & messing up its installation.  Much happier with the prospect of using a probe without that hassle.
